Snippets/Castle/Extensions.DependencyInjection_1/Usage.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Castle.MicroKernel.Registration;
+using Castle.Windsor.MsDependencyInjection;
+using NServiceBus;
+
+class Usage
+{
+    Usage(EndpointConfiguration endpointConfiguration)
+    {
+        #region CastleWindsor
+
+        var containerSettings = endpointConfiguration.UseContainer(new WindsorServiceProviderFactory());
+
+        containerSettings.ConfigureContainer(c => c.Register(Component.For<MyService>()
+                .Instance(new MyService())));
```

Comment:
the line break is a bit weird here imo. I'd suggest moving the registration part onto a single line